config/webpack/custom.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    resolve: {
+        extensions: ['.css', '.scss']
```

Comment:
Do we need to support both file extensions?

Reply:
Done. custom.js not needed anymore